Clinical-Genomics/scout
# Single other causative

This PR adds a functionality or fixes a bug.

Show other causative once, even if several events point to it (and obviously keep checking that it still is causative).

**How to test**:
1. mark a variant present in two cases causative twice in one case
2. notice how it shows twice in the other causatives list from the other case
3. apply patch and find it only once

![Screenshot 2021-03-30 at 09 23 13](https://user-images.githubusercontent.com/758570/112951433-a27b0000-913b-11eb-89c7-c3b0b66e66c4.png)
![Screenshot 2021-03-30 at 09 25 45](https://user-images.githubusercontent.com/758570/112951437-a3ac2d00-913b-11eb-90db-310bcea12e4b.png)


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
